internal/clients/stubs/ec2_stub.go

```diff
@@ -41,7 +44,15 @@ func (mock *EC2ClientStub) Status(ctx context.Context) error {
 }
 
 func (mock *EC2ClientStub) ImportPubkey(ctx context.Context, key *models.Pubkey, tag string) (string, error) {
-	return "", nil
+	mock.Imported = append(mock.Imported, key)
+	return "key-12345678912345678", nil
+}
+
+func (mock *EC2ClientStub) GetPubkeyName(ctx context.Context, fingerprint string) (string, error) {
+	if fingerprint == "existing" {
```

Comment:
Highly encourage to use proper data than fake one :-) Like proper fingerprint of a real key... :-)

Reply:
Well it is a special value at this moment, so I wanted to make it extra clear it is a special value. Real value might confuse ppl where it is defined. With real data, I'd want the key to actually be in the mock (added).

But that might be cleanest approach, I was just lazy to define fixtures for ex2, I needed to write a lot of setup code for tests already :) but if we want that, I'm happy to work on it.

Reply:
I do not understand why you want to use this "special" value. Can't you simply update the stub to actually accept real Pubkey entries (with real data) so you can then get pubkey by name using it's real fingerprint? We already have several real pubkeys in the test suite (mine, Anna's...) and there is nothing wrong with creating stubs that actually do stuff. I actually thing although it will take more time, it will pay off as soon as it gets more complex later on.